### PR TITLE
fix tooltip on badges in topology

### PR DESCRIPTION
**Pull request by invincibleJai**
fix tooltip on badges in topology

Tracks: https://jira.coreos.com/browse/ODC-1539

### CODE REVIEW

File: frontend/packages/dev-console/src/components/topology/shapes/ResourceIcon.scss [hunk 1]
```diff
@@ -9,5 +9,6 @@
 .odc-resource-icon-deployment,
 .odc-resource-icon-daemonset,
 .odc-resource-icon-deploymentconfig {
+  cursor: pointer;
```
Comment (vikram-raj) [inline frontend/packages/dev-console/src/components/topology/shapes/ResourceIcon.scss]: This should not be a cursor pointer. And nothing should happen when the user clicks on the node name. The only node should be clickable. Can you confirm this with UX?

File: frontend/packages/dev-console/src/components/topology/shapes/BaseNode.scss [hunk 2]
```diff
@@ -10,7 +10,7 @@
   }
 
   &__label {
-    pointer-events: none;
+    cursor: default;
```
Comment (vikram-raj) [inline frontend/packages/dev-console/src/components/topology/shapes/BaseNode.scss]: removing ```pointer-events: none;``` makes nodes name clickable and it's open the overview page.  I am not sure it is expected behavior. cc: @serenamarie125

Comment (joshuawilson) [reply frontend/packages/dev-console/src/components/topology/shapes/BaseNode.scss]: I kind of liked that I could open the details from the name thing.

File: frontend/packages/dev-console/src/components/topology/shapes/ResourceIcon.scss (hunk 1 above)
Comment (invincibleJai) [reply frontend/packages/dev-console/src/components/topology/shapes/ResourceIcon.scss]: Sorry @vikram-raj saw it now. Only icon has pointer effect i.e `D`, `DC` etc not the node name as we have in other badges for resources. Will check on this with UXD

@openshift/team-devconsole-ux

File: frontend/packages/dev-console/src/components/topology/shapes/BaseNode.scss (hunk 2 above)
Comment (invincibleJai) [reply frontend/packages/dev-console/src/components/topology/shapes/BaseNode.scss]: @vikram-raj got your point but if we'll have `pointer-events: none;` then how will address above issue of showing tooltip for resource badges [0] as with above even hover won't get triggered. 

Will file a new PR as this is merged once decided.

[0] https://jira.coreos.com/browse/ODC-1539

Comment (vikram-raj) [reply frontend/packages/dev-console/src/components/topology/shapes/BaseNode.scss]: @invincibleJai Currently, everything in node wrap under a single ```<g>``` tag and ```onClick``` event to open the overview added to it. If we can move label from that ```<g>``` tag put it in another ```<g>``` tag then we can resolve this issue.

**Pull request merged**